rosdep/python.yaml

```diff
@@ -1134,6 +1134,12 @@ python-gi:
   fedora: [pygobject3]
   gentoo: [dev-python/pygobject]
   ubuntu: [python-gi]
+python-git:
+  arch: [python2-gobject]
+  debian: [python-git]
+  fedora: [GitPython]
+  gentoo: [git-python]
```

Comment:
Shouldn't this package name be prefixed with `dev-python/` per https://packages.gentoo.org/packages/dev-python/git-python (currently offline)

Reply:
I did not know that, thanks. Updated.